testutil/ciao-down/ciao_down.go

```diff
@@ -64,10 +64,57 @@ func init() {
 	}
 }
 
-func vmFlags(fs *flag.FlagSet, memGB, CPUs *int) {
+type mounts []mount
+
+func (m *mounts) String() string {
+	return fmt.Sprint(*m)
+}
+
+func (m *mounts) Set(value string) error {
+	components := strings.Split(value, ",")
+	if len(components) != 3 {
+		return fmt.Errorf("--mount parameter should be of format tag,security_model,path")
+	}
+	*m = append(*m, mount{
+		Tag:           components[0],
+		SecurityModel: components[1],
+		Path:          components[2],
+	})
+	return nil
+}
+
+type ports []portMapping
+
+func (p *ports) String() string {
+	return fmt.Sprint(*p)
+}
+
+func (p *ports) Set(value string) error {
+	components := strings.Split(value, "-")
```

Comment:
My only feedback is that I think <host>:<guest> is a more common notation to refer to port mappings rather than a hyphen.

But that's not enough to hold this PR back :-)